clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/client/TaskCacheClient.java

```diff
@@ -30,11 +32,14 @@
 
 import static com.netflix.spinnaker.clouddriver.ecs.cache.Keys.Namespace.TASKS;
 
+@Component
 public class TaskCacheClient extends AbstractCacheClient<Task> {
+  private ObjectMapper mapper;
 
   @Autowired
-  public TaskCacheClient(Cache cacheView) {
+  public TaskCacheClient(Cache cacheView, @Qualifier("objectMapper") ObjectMapper mapper) {
```

Comment:
there should be no need for a @Qualifier annotation.  This is to be used when multiple object mappers are available, which should NOT be the case in clouddriver.  Besides, having to use the @Qualifier annotation is a very strong code smell

Reply:
As discussed, this is a lesser evil.